catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1273,7 +1274,9 @@ public UpdateResponse update(UpdateRequest updateRequest)
             queryImpl.setStartIndex(1);
             queryImpl.setPageSize(updateReq.getUpdates()
                     .size());
-            QueryRequestImpl queryRequest = new QueryRequestImpl(queryImpl);
+            QueryRequestImpl queryRequest = new QueryRequestImpl(queryImpl,
+                    updateReq.getStoreIds());
```

Comment:
Most of the time there will be 0 storeIds and the method will return an empty set (default behaviour). In the case of an empty set it will use the local provider by default.